client/app/components/tags-control/EditTagsDialog.jsx

```diff
@@ -9,6 +9,21 @@ function EditTagsDialog({ dialog, tags, getAvailableTags }) {
   const [availableTags, setAvailableTags] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
   const [values, setValues] = useState(() => uniq(map(tags, trim))); // lazy evaluate
+  const [selectRef, setSelectRef] = useState(null);
+
+  // For some reason `Select::autoFocus` doesn't work in dialogs.
+  // Workaround: get component's ref and manually pass focus
```

Comment:
Looks like adding `destroyOnClose` to the dialog fixes it -> [example](https://codesandbox.io/s/spring-dust-v6t0v)

Reply:
In this case it doesn't. We use dialogs in a bit different way - we mount them with `ReactDOM.render()` and it's destroyed when closed. I think `autoFocus` may not work because of this.

Reply:
True, in this case it indeed doesn't work for a different reason 😂. Even that workaround didn't work to me when I tested here now.

The `Select` starts as `disabled`, the autoFocus works, but not when it's disabled. So we either have to remove that disabled prop (then we can use autoFocus prop) or make the focus happen only when the loading is complete (using the ref).

Reply:
@gabrieldutra thank you for pointing! Please check https://github.com/getredash/redash/pull/4979/commits/9ad3ba1a7381cde95505395bb5f9bcabacda3f58